h/cli/commands/publisher.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+
+import click
+
+from h import models
+
+
+@click.group()
+def publisher():
+    """Manage publisher accounts."""
+
+
+@publisher.command()
+@click.option('--name', prompt=True, help="The name of the publisher")
+@click.option('--authority', prompt=True, help="The authority (domain name) for the publisher")
+@click.pass_context
+def add(ctx, name, authority):
```

Comment:
It might be helpful to display a warning message at the beginning stating that this CLI command is only intended for development environments. At least until we can create groups without creators, because that admin user is just a really nasty hack to make it work when I sent you the code snippet.